src/libraries/System.Private.Uri/src/System/Uri.cs

```diff
@@ -5185,5 +5185,15 @@ internal bool HasAuthority
                 return InFact(Flags.AuthorityFound);
             }
         }
+
+        static bool IParsable<Uri>.TryParse(string? s, IFormatProvider? provider, out Uri result)
```

Comment:
The nullability can be handled by adding `[MaybeNullWhen(returnValue: false)]` to the `out Uri result` parameter.

The return value should match whether `result` is null or not.